build.rs

```diff
@@ -405,6 +405,10 @@ const MODULE_RAW_LINES: &'static [(&'static str, &'static str)] = &[
 /// Rerun this build script if files under mozjs/ changed, unless this returns true.
 /// Keep this in sync with .gitignore
 fn ignore(path: &Path) -> bool {
+    if path.is_dir() {
```

Comment:
Missing `!`?

Reply:
Nope, I specifically don't care about directories because Python puts some `pyc` files in them and so Cargo thinks they are stale.

Reply:
Might be worth adding a comment to that effect?

Reply:
Well there is a comment here now! Just kidding, I'll add one. :)